lib/core.ps1

```diff
@@ -18,6 +18,13 @@ $globaldir = $env:SCOOP_GLOBAL, "$env:ProgramData\scoop" | select -first 1
 #       Use at your own risk.
 $cachedir = $env:SCOOP_CACHE, "$scoopdir\cache" | select -first 1
 
+# Overwrite $scoopdir, $globaldir and $cachedir on unix systems
```

Comment:
How about adding these overrides in auto-pr.ps1 and checkver.ps1 after core.ps1 has been dot sourced? It would be good to keep core.ps1 clean.